### ISSUE TITLE
Install on conda distrubution

### ISSUE THREAD

**Issue by cpashartis** [libAtoms/extxyz#13]
Hello,

For some reason, I am unable to install extxyz by simply
```pip install extxyz```
I get the error: 
```
ERROR: Could not find a version that satisfies the requirement extxyz (from versions: none)
ERROR: No matching distribution found for extxyz
```

I then tried to ```pip install git+https://github.com/libAtoms/extxyz```
but this subsequently fails looking for the ftp server for ftp.pcre.org. This also seems to be the case when I try to use the setup.py from the source code.

Is anyone else having this issue or knows how to resolve it? My pip is perfectly fine as I installed packages just before it, the only added item I can give is I am on WSL2.

**Comment by jameskermode** [comment]
Thanks for reporting, I'll take a look. We don't yet have binary wheels for Windows, but I would have expected WSL2 to work with the manylinux wheels. The failure to build from source is worrying as you say.

**Comment by jameskermode** [comment]
Ah, I think I see the problem. What version of Python are you using? We haven't made a release in some time, so Python 3.10 and 3.11 are not yet supported. I'll enable them in the CI and make a new release soon.

**Comment by cpashartis** [comment]
Yeah, perhaps I should have mentioned I'm using python 3.10. Let me know
when you manage to update it accordingly :)


Thanks,

Christopher Pashartis

**Comment by jameskermode** [comment]
Resolved with release of extxyz==0.3.0 and later
